src/scalper_utils.ts

```diff
@@ -307,4 +310,67 @@ export async function getFairValue(connection, spotMarket, symbol) {
     }
   }
   return fairValue;
+}
+
+export async function jupiterHedge(connection: Connection, owner: Keypair, hedgeSide: string, 
+    base: string, quote: string, hedgeDelta: number, hedgePrice: number) {
+  const jupiter = await Jupiter.load({
+    connection: connection,
+    cluster: cluster,
+    user: owner, 
+    wrapUnwrapSOL: false, 
+  });
+  let inputToken: PublicKey;
+  let outputToken: PublicKey;
+  let hedgeAmount: number;
+  if (hedgeSide == "sell"){
```

Comment:
make hedgeSide be an enum

Reply:
Used a string everywhere else. If not necessary I worry that changing all the references I may mess something up in well tested code

Reply:
Make a TODO and I will swap it out in a follow up.
If you swap it to an enum, then I think type checking should actually catch the references for you.